Framework/Algorithms/src/ReflectometryReductionOneAuto.cpp

```diff
@@ -520,11 +520,47 @@ bool ReflectometryReductionOneAuto::checkGroups() {
   }
   return false;
 }
+/**
+ * Sum over transmission group workspaces to produce one
+ * workspace.
+ * @param transGroup : The transmission group to be processed
+ * @return A workspace pointer containing the sum of transmission workspaces.
+ */
+Mantid::API::Workspace_sptr
+ReflectometryReductionOneAuto::sumOverTransmissionGroup(
+    WorkspaceGroup_sptr &transGroup) {
+  // Handle transmission runs
+  auto transmissionRunSum = transGroup->getItem(0);
+  MatrixWorkspace_sptr total;
+  for (size_t item = 1; item < transGroup->size(); ++item) {
+    auto plusAlg = this->createChildAlgorithm("Plus");
```

Comment:
as just discussed, it is probably good to create the algorithm only once and only reasign/execute in a loop. Maybe time the difference, that would be interesting